openapi_python_client/__init__.py

```diff
@@ -72,6 +76,8 @@ def __init__(self, *, openapi: OpenAPI) -> None:
         self.package_dir: Path = self.project_dir / self.package_name
         self.package_description = f"A client library for accessing {self.openapi.title}"
 
+        self.env.filters.update(self.TEMPLATE_FILTERS)
```

Comment:
Awesome!  I only ever use the basics of jinja, didn't know you could add filters like this.